end_to_end/0-AutoClaimFraudDetection.ipynb

```diff
@@ -0,0 +1,1848 @@
+{
```

Comment:
<p>In line 10, <code>df</code> is not defined.</p><p> </p><p> </p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
&lt;ipython-input-18-b7cf35d89be5&gt; in &lt;module&gt;
      8            'incident_severity','fraud']
      9 
---&gt; 10 corr_df = df[corr_list]
     11 corr=round(corr_df.corr(),2)
     12 
 
NameError: name 'df' is not defined
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
pls replace with df_combined

Reply:
fix to `df_combined`